CHANGELOG_PENDING.md

```diff
@@ -21,6 +21,8 @@
 
 #### General
 
+- \#1859 Adds check for invalid inferred IP (@reubenr0d)
```

Comment:
Looks good , just one small detail here. This change only impacts the orchestrator so let's change this to be under "Orchestrator"

Perhaps we could also update the text to be a little more descriptive, maybe something like "Handle error for invalid inferred orchestrator public IP address on node startup"